packages/k2-alpine/package.json

```diff
@@ -26,7 +27,9 @@
     "expo-status-bar": "1.12.1",
     "react": "18.3.1",
     "react-native": "0.73.7",
+    "react-native-gesture-handler": "2.14.1",
     "react-native-reanimated": "3.6.2",
+    "react-native-reanimated-carousel": "v4.0.0-canary.22",
```

Comment:
introduced for implementing infinite scrolling. I had to modify it slightly to allow touch events outside the item layout for implementing our hexagonal multi-row layout. I [opened a pr](https://github.com/dohooo/react-native-reanimated-carousel/pull/733) and they [merged it quickly](https://github.com/dohooo/react-native-reanimated-carousel/releases/tag/v4.0.0-canary.21).